<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lismpeg</class>
 <widget class="QDialog" name="lismpeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert screenshots to mpeg</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/openlisem.qrc">
    <normaloff>:/2X/film.png</normaloff>:/2X/film.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mencoder lcation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="E_mencoderDir"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton_mencoderDir">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/openlisem.qrc">
       <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Result folder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="E_resultsDir"/>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_resultDir">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/openlisem.qrc">
       <normaloff>:/2X/Folder-icon.png</normaloff>:/2X/Folder-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QToolButton" name="toolButton_createMP4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Create mp4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="E_mpegProcessOutput"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/openlisem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lismpeg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lismpeg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
